.circleci/config.yml

```diff
@@ -65,8 +69,22 @@ jobs:
       MINIKUBE_WANTUPDATENOTIFICATION: false
       MINIKUBE_WANTREPORTERRORPROMPT: false
       SKAFFOLD_VERSION: v0.18.0
+      GO111MODULE: "on"
+      GOVERSION: 1.12.5
+      OS: linux
+      ARCH: amd64
     steps:
       - checkout
+      - run:
+          name: Install go
+          command: |
+            go version
+            go env GOROOT
+            mkdir tmp
+            cd tmp
+            sudo rm -rf /usr/local/go
+            wget https://storage.googleapis.com/golang/go$GOVERSION.$OS-$ARCH.tar.gz
```

Comment:
Why are we installing go?

Reply:
I think the machine image apparently has an older version of Go? Maybe we could cache, or use a newer machine image.

Reply:
I don't know how that's possible because this is a container image.